package.json

```diff
@@ -45,11 +45,14 @@
     "after": "~0.8.1",
     "browzers": "1.2.0",
     "bulk-require": "0.2.1",
+    "dependency-check": "^2.10.0",
+    "electron": "^1.8.2",
+    "electron-prebuilt": "^1.4.13",
     "mocha": "~1.16.2",
+    "phantomjs": "^2.1.7",
```

Comment:
I think this one should be ignored as well because `phantomjs-prebuilt` is the preferred package; `phantomjs` is legacy.

Reply:
But we intend to remove phantomjs anyway, so doesn't matter much.

Reply:
Aah this is better, I'll ignore `phantomjs` instead, also `electron`.